stdlib/asyncio/windows_events.pyi

```diff
@@ -48,6 +48,12 @@ if sys.platform == "win32":
         def select(self, timeout: int | None = None) -> list[futures.Future[Any]]: ...
         def recv(self, conn: socket.socket, nbytes: int, flags: int = 0) -> futures.Future[bytes]: ...
         def recv_into(self, conn: socket.socket, buf: WriteableBuffer, flags: int = 0) -> futures.Future[Any]: ...
+        def recvfrom(
+            self, conn: socket.socket, nbytes: int, flags: int = 0
+        ) -> futures.Future[tuple[int, socket._RetAddress]]: ...
```

Comment:
```suggestion
        ) -> futures.Future[tuple[bytes, socket._RetAddress]]: ...
```
There is an error case returning bytes in the Python code, and if I'm reading the C code in `_overlapped` correctly, this hits the `TYPE_READ_FROM` case in `_overlapped_Overlapped_getresult_impl`, which returns a bytes in the first tuple member.

Reply:
Good catch.

Reply:
That matches the other `recvfrom` signatures in typeshed as well. I think I misread some things somewhere.